src/Umbraco.Web.UI.Client/src/common/services/search.service.js

```diff
@@ -119,7 +113,7 @@ angular.module('umbraco.services')
 
                 return entityResource.searchAll(args.term, args.canceler).then(function (data) {
 
-                    _.each(data, function (resultByType) {
+                    data.forEach(resultByType => {
```

Comment:
@nathanwoulfe Uh oh! This one seems to have broken! 

![image](https://user-images.githubusercontent.com/304656/89017522-b7886080-d31a-11ea-96bb-477e611da75a.png)


Little help please? 🙏

Reply:
Reproduce by trying to search in the backoffice:

![image](https://user-images.githubusercontent.com/304656/89017596-d5ee5c00-d31a-11ea-8fbd-df716856c977.png)

Reply:
Hmm. I thought I'd sorted that... I'm guessing data isn't an array, but an object/dictionary. I'll grab the laptop...